s1aptests/test_multi_enb_partial_reset failure

### Your Environment

- **Version:** HEAD
- **Affected Component:** S1APTest
- **Affected Subcomponent:** s1ap_utils.py
- **Deployment Environment:** circleCI

### Describe the Issue

```
[magma@10.240.0.45] out: Running test: s1aptests/test_multi_enb_partial_reset.py
[magma@10.240.0.45] out: timeout --foreground -k 930s 900s sudo -E PATH=/home/vagrant/build/python/bin:/home/vagrant/build/python/bin:/usr/lib/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin PYTHONPATH=:/home/vagrant/s1ap-tester/bin:/home/vagrant/s1ap-tester/bin /home/vagrant/build/python/bin/nosetests --with-xunit --xunit-file=/var/tmp/test_results/test_multi_enb_partial_reset.xml -x -s s1aptests/test_multi_enb_partial_reset.py || exit 1
[magma@10.240.0.45] out: Start time 23:46:10
[magma@10.240.0.45] out: tag: AUTH_TYPE tagVal: MILENAGE
[magma@10.240.0.45] out: ************************* Enb tester config
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out: **********************TPT open server******** 0
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out:  Server Socket FD =[13]
[magma@10.240.0.45] out: ************* Testing *Time elapsed(in usec): for timer expiry               :161 usec
[magma@10.240.0.45] out: AGW is stateless
[magma@10.240.0.45] out: APN Correction configured
[magma@10.240.0.45] out: Health service is disabled
[magma@10.240.0.45] out: ***************** Sending Multiple Enb Config Request
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out: **** data->numOfEnbs = 5
[magma@10.240.0.45] out: Using subscriber IMSI IMSI001010000000001
[magma@10.240.0.45] out: Using IMEI 3805468432113171
[magma@10.240.0.45] out: ************************* UE device config for ue_id  1
[magma@10.240.0.45] out: ************************* Configuring IP block
[magma@10.240.0.45] out: ************************* Waiting for IP changes to propagate
[magma@10.240.0.45] out: ************************* S1 setup
[magma@10.240.0.45] out: ************************* UE App config
[magma@10.240.0.45] out: ************************* Calling attach for UE id  1
[magma@10.240.0.45] out: ue id 1 found
[magma@10.240.0.45] out: PDN_TYPE: 1
[magma@10.240.0.45] out: IPv4 PDN_ADDRESS:192 168 128 12
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out: *************NbUiNbuHdlUeRadioCapMsg: Send UE cap msg SUCCESS*********
[magma@10.240.0.45] out: ************************* Sending eNB Partial Reset Request
[magma@10.240.0.45] out: Reset_req.r.partialRst.ueS1apIdPairList[indx].ueId 1 0
[magma@10.240.0.45] out: ue_ids [1]
[magma@10.240.0.45] out: Deleting Ue Context from Traffic Handler
[magma@10.240.0.45] out: ************************* Calling detach for UE id  1
[magma@10.240.0.45] out: ************************* send SCTP SHUTDOWN
[magma@10.240.0.45] out: Keys left in Redis (list should be empty)[
[magma@10.240.0.45] out:   
[magma@10.240.0.45] out: ]
[magma@10.240.0.45] out: Entries in s1ap_imsi_map (should be zero): 0
[magma@10.240.0.45] out: Entries left in hashtables (should be zero): 0
[magma@10.240.0.45] out: Entries in mme_ueip_imsi_map (should be zero): 1
[magma@10.240.0.45] out: F
[magma@10.240.0.45] out: ======================================================================
[magma@10.240.0.45] out: FAIL: Multi eNB + attach 1 UE + s1ap partial reset + detach
[magma@10.240.0.45] out: ----------------------------------------------------------------------
[magma@10.240.0.45] out: Traceback (most recent call last):
[magma@10.240.0.45] out:   File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/test_multi_enb_partial_reset.py", line 115, in test_multiple_enb_partial_reset
[magma@10.240.0.45] out:     ue, s1ap_types.ueDetachType_t.UE_NORMAL_DETACH.value, True,
[magma@10.240.0.45] out:   File "/home/vagrant/magma/lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py", line 390, in detach
[magma@10.240.0.45] out:     == response.msg_type
[magma@10.240.0.45] out: AssertionError: 
[magma@10.240.0.45] out: -------------------- >> begin captured logging << --------------------
[magma@10.240.0.45] out: root: ERROR: Could not load magmad yml config for mconfig modules
[magma@10.240.0.45] out: urllib3.connectionpool: DEBUG: Starting new HTTP connection (1): 192.168.60.142:3333
[magma@10.240.0.45] out: urllib3.connectionpool: DEBUG: http://192.168.60.142:3333 "GET /stats/switches HTTP/1.1" 200 17
[magma@10.240.0.45] out: root: INFO: Adding subscriber : IMSI001010000000001
[magma@10.240.0.45] out: root: DEBUG: s1ap response expected, received: 12, 12
[magma@10.240.0.45] out: root: DEBUG: s1ap message expected, received: 68, 68
[magma@10.240.0.45] out: root: INFO: Deleting subscriber : IMSI001010000000001
[magma@10.240.0.45] out: --------------------- >> end captured logging << ---------------------
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out: ----------------------------------------------------------------------
[magma@10.240.0.45] out: Ran 1 test in 8.704s
[magma@10.240.0.45] out: 
[magma@10.240.0.45] out: FAILED (failures=1)
```

Examples:
https://app.circleci.com/pipelines/github/magma/magma/31572/workflows/ae34e44a-3ff7-464a-9a3c-7387eeb99a50/jobs/364878
https://app.circleci.com/pipelines/github/magma/magma/31519/workflows/7d78a3d8-d754-43e9-9f9b-ea43d127293c/jobs/363919

--- a/lte/gateway/python/integ_tests/s1aptests/test_multi_enb_partial_reset.py
+++ b/lte/gateway/python/integ_tests/s1aptests/test_multi_enb_partial_reset.py
@@ -106,6 +106,7 @@ class TestMultipleEnbPartialReset(unittest.TestCase):
         )
         response = self._s1ap_wrapper.s1_util.get_response()
         self.assertEqual(response.msg_type, s1ap_types.tfwCmd.RESET_ACK.value)
+        time.sleep(0.5)
         # Trigger detach request
         for ue in ue_ids:
             print("************************* Calling detach for UE id ", ue)

